foris/locale/de/LC_MESSAGES/foris.po

```diff
@@ -1167,8 +1170,8 @@ msgid ""
 "will also not be able to install Updater's package lists."
 msgstr ""
 "Ohne den Updater-Dienst wird die Installierte Software nicht aktualisiert. "
-"Es wird auch nicht möglich sein, Listen der Softwarepakete, die zu Updater-"
-"Dienst gehören, zu installieren."
+"Darüber hinaus ist es auch nicht möglich Software aus der Liste im Updater-"
```

Comment:
I would prefer a future tense here:
'Darüber hinaus wird es auch nicht möglich sein Software aus der Liste im Updater-Abschnitt zu installieren.'

The reason is that if the user keeps the service disabled he won't be able to install the software (in future).

My understanding of the form with the present tense is that the user won't be able to install the software regardless of whether he chose to activate/deactivate the service. This may be caused by the fact that for me 'darüber hinaus' has the same meaning as 'außerdem'.